ui/js/component/incompatibleDaemonModal/view.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Modal } from "component/modal";
+
+class IncompatibleDaemonModal extends React.PureComponent {
+  render() {
+    const { quit, skipWrongDaemonNotice } = this.props;
+
+    return (
+      <Modal
+        isOpen={true}
+        contentLabel={__("Incompatible daemon running")}
+        type="confirm"
+        confirmButtonLabel={__("Exit")}
+        abortButtonLabel={__("Continue")}
+        onConfirmed={quit}
+        onAborted={skipWrongDaemonNotice}
+      >
+        {__(
+          "An incompatible version of the LBRY daemon is running. If you continue to receive this message after starting LBRY again, try restarting your computer."
```

Comment:
Let's make this a link to a KB page, so we can manage the process and provide more information:

An incompatible version of the LBRY daemon is running. Please see [this article](https://lbry.io/faq/incompatible-daemon on how to fix this problem.

@tzarebczan please create a KB article at `lbry.io/faq/incompatible-daemon` on what users should do when their daemon is out of date (I think it's basically kill all processes and/or restart and if that doesn't fix re-install.)